designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/model/LocationItemModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.hisp.dhis.mobile.ui.designsystem.component.model
+
+sealed class LocationItemModel(
```

Comment:
Could you add documentation for this model?

Reply:
Done!